experimental/dds/tree2/src/feature-libraries/fence.json

```diff
@@ -1,5 +1,13 @@
 {
 	"tags": ["feature-libraries"],
 	"exports": ["index"],
-	"imports": ["codec", "events", "util", "core", "shared-tree-core", "external-utilities", "codec"]
+	"imports": [
+		"codec",
+		"events",
+		"util",
+		"core",
+		"shared-tree-core",
+		"external-utilities",
```

Comment:
We specifically should not take this dependency. external-utilities should only ever be imported by users of this package, not by anythin in the package itself. They are seperate for package size reasons, and should have a readme with more details in it if you are interested.

Reply:
removed "external-utilities"

Reply:
its still here.

Reply:
removed "external-utilities"